src/app/shared-utils/form-utils/input-field-email/input-field-email.component.spec.ts

```diff
@@ -0,0 +1,56 @@
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+import {FormsModule} from '@angular/forms';
+
+import {InputFieldEmailComponent} from './input-field-email.component';
+
+describe('InputFieldEmailComponent', () => {
+  let component: InputFieldEmailComponent;
+  let fixture: ComponentFixture<InputFieldEmailComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [FormsModule],
+      declarations: [InputFieldEmailComponent]
+    })
+      .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(InputFieldEmailComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should initialize the component', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should define input element ', () => {
+    const element = fixture.debugElement.nativeElement;
+    expect(element.querySelector('input')).toBeDefined();
+  });
+
+  it('should enter an email address', () => {
+    const element = fixture.debugElement.nativeElement;
+    element.querySelector('input').value = 'rowansiwakoti@gmail.com';
+    fixture.detectChanges();
+    expect(element.querySelector('input').value).toEqual('rowansiwakoti@gmail.com');
+  });
+
+  it('should enter an invalid email address', () => {
```

Comment:
write a proper description in a suite. Similar to should alert error message when an email address is invalid.